lib/dotdiff/snapshot.rb

```diff
@@ -54,10 +54,12 @@ def diff_file
       File.join(failure_path, "#{base_filename(false)}.diff.#{IMAGE_EXT}")
     end
 
-    def capture_from_browser(hide_and_show = true, element_handler = ElementHandler.new(page))
+    def capture_from_browser(hide_and_show)
       return fullscreen_file if use_custom_screenshot
 
       if hide_and_show
+        element_handler = ElementHandler.new(page)
```

Comment:
Shame this is no longer dependency injected into the method, however looking seems like it wasn't actually overridden anywhere so don't have strong opinion on this.